### ISSUE TITLE
Feature Proposal: method to retrieve ImagePartType from file extension

### ISSUE THREAD

**Issue by JaimeStill** [OfficeDev/Open-XML-SDK#1081]
**Description**

In working through some abstractions for working with images in wordprocessing documents, I wrote a method for getting the `ImagePartType` based on the provided extension. I'm trying to see the utility of incorporating this into the API, and if so, where I might incorporate it?

```cs
// proposed method
static ImagePartType GetImagePartType(string ext) =>
    ext.ToLower() switch
    {
        ".bmp"  => ImagePartType.Bmp,
        ".emf"  => ImagePartType.Emf,
        ".ico"  => ImagePartType.Icon,
        ".jpg"  => ImagePartType.Jpeg,
        ".jpeg" => ImagePartType.Jpeg,
        ".pcx"  => ImagePartType.Pcx,
        ".png"  => ImagePartType.Png,
        ".svg"  => ImagePartType.Svg,
        ".tiff" => ImagePartType.Tiff,
        ".wmf"  => ImagePartType.Wmf,
        _ => throw new NotSupportedException($"{ext} is not supported")
    };

// example usage
static ImagePart LoadImageIntoDoc(MainDocumentPart main, FileInfo file)
{
    var imagePart = main.AddImagePart(GetImagePartType(file.Extension));
    // remaining method excluded for brevity
}
```

**Comment by rmboggs** [comment]
I think this is a good idea.  I'm not sure where this would fit in this project though.

**Comment by JaimeStill** [comment]
@rmboggs capabilities similar to this (but inverse) exist in an [`ImagePartTypeInfo`](https://github.com/OfficeDev/Open-XML-SDK/blob/main/src/DocumentFormat.OpenXml/Packaging/ImagePartTypeInfo.cs) class, but it is not part of the public facing API.

**Comment by tomjebo** [comment]
@JaimeStill That looks very doable, please feel free to submit a PR and we can iterate on it if needed.

**Comment by JaimeStill** [comment]
Closed by #1082

**Comment by Asbjoedt** [comment]
Hi
This is interesting. Of course the list of image extensions are way longer than the one here, and you can't add everyone, but I would like this one added.
```".tif" => ImagePartType.Tiff,```

I also saw .gif is missing.
```".gif" => ImagePartType.Gif,```

I can make a PR myself (my first) that adds these two line.
